Tests/Unit/TwitterTests.m

```diff
@@ -622,4 +622,67 @@ - (void)testAuthorizeWithoutLocalAccountAndNetworkSuccess {
     OCMVerifyAll(mockedURLSession);
 }
 
+- (void)testDeauthorizeLoggedOutAccount {
+    id store = PFStrictClassMock([ACAccountStore class]);
+    NSURLSession *session = PFStrictClassMock([NSURLSession class]);
+    id mockedDialog = PFStrictProtocolMock(@protocol(PFOAuth1FlowDialogInterface));
+    PF_Twitter *twitter = [[PF_Twitter alloc] initWithAccountStore:store urlSession:session dialogClass:mockedDialog];
+
+    XCTestExpectation *expectation = [self currentSelectorTestExpectation];
+    [[twitter authorizeInBackground] continueWithBlock:^id(BFTask *task) {
+        NSError *error = task.error;
+        XCTAssertNotNil(error);
+        XCTAssertEqualObjects(error.domain, PFParseErrorDomain);
+        XCTAssertEqual(error.code, 2);
```

Comment:
oops! a previous iteration of this commit added a new error code of `2`, and I missed this in cleaning it up 😅